pkg/exchange/binance/stream.go

```
@@ -132,12 +152,28 @@ func NewStream(ex *Exchange, client *binance.Client, futuresClient *futures.Clie
 
 	stream.OnDisconnect(stream.handleDisconnect)
 	stream.OnConnect(stream.handleConnect)
-	stream.OnListenKeyExpired(func(e *ListenKeyExpired) {
-		stream.Reconnect()
-	})
 	return stream
 }
 
+func prasePrivateKey(pemString string) (ed25519.PrivateKey, error) {
+	block, _ := pem.Decode([]byte(pemString))
```

error is not checked

---

The second return value of `pem.Decode` is not `error`.

---

It's the resting bytes not yet decoded.

---

got it~